automatic/graphviz/graphviz.nuspec

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata>
+    <id>graphviz</id>
+    <version>0.0</version>
+    <title>graphviz</title>
+    <authors>Arif Bilgin, graphviz committers</authors>

```

Comment:
I will have to check with the other maintainers, but I have never heard of Arif Bilgin. `git grep` for this person in the Graphviz repo returns nothing. Though I note this is indeed one of the people listed on nuget.org for this package. Or maybe this is you, @RedBaron2? I'm not sure if this field is meant to be upstream authors or authors of the Chocolatey package.

Reply:
Maybe one of the maintainers can answer this question

Reply:
It turns out Arif Bilgin was one of the original authors, just from before my time, back in the AT&T days. You can ignore my previous comment.